deployments/deployment/nginx-plus-ingress.yaml

```diff
@@ -82,3 +92,30 @@ spec:
          #- -enable-prometheus-metrics
          #- -enable-service-insight
          #- -global-configuration=$(POD_NAMESPACE)/nginx-configuration
+#      securityContext:
+#        fsGroup: 101 #nginx
```

Comment:
I realise the new `securityContext` block that was added in https://github.com/nginxinc/kubernetes-ingress/pull/3573 is conflicting with this one.  We can remove this `securityContext` and move the `fsGroup: 101 #nginx` line into the new on Line 23 of this file.

https://github.com/nginxinc/kubernetes-ingress/blob/09601f457eda4f5126f1c63baabe314b3e633b98/deployments/deployment/nginx-plus-ingress.yaml#L23-L26

Reply:
Indeed - applied to the manifests as well as Helm chart!